core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -23,86 +23,26 @@
 import org.apache.carbondata.core.util.CarbonProperty;
 
 public final class CarbonCommonConstants {
-  /**
-   * surrogate value of null
-   */
-  public static final int DICT_VALUE_NULL = 1;
-  /**
-   * surrogate value of null for direct dictionary
-   */
-  public static final int DIRECT_DICT_VALUE_NULL = 1;
-  /**
-   * integer size in bytes
-   */
-  public static final int INT_SIZE_IN_BYTE = 4;
-  /**
-   * short size in bytes
-   */
-  public static final int SHORT_SIZE_IN_BYTE = 2;
-  /**
-   * DOUBLE size in bytes
-   */
-  public static final int DOUBLE_SIZE_IN_BYTE = 8;
-  /**
-   * LONG size in bytes
-   */
-  public static final int LONG_SIZE_IN_BYTE = 8;
-  /**
-   * byte to KB conversion factor
-   */
-  public static final int BYTE_TO_KB_CONVERSION_FACTOR = 1024;
-  /**
-   * BYTE_ENCODING
-   */
-  public static final String BYTE_ENCODING = "ISO-8859-1";
-  /**
-   * measure meta data file name
-   */
-  public static final String MEASURE_METADATA_FILE_NAME = "/msrMetaData_";
-
-  /**
-   * set the segment ids to query from the table
-   */
-  public static final String CARBON_INPUT_SEGMENTS = "carbon.input.segments.";
-
-  /**
-   * key prefix for set command. 'carbon.datamap.visible.dbName.tableName.dmName = false' means
-   * that the query on 'dbName.table' will not use the datamap 'dmName'
-   */
-  @InterfaceStability.Unstable
-  public static final String CARBON_DATAMAP_VISIBLE = "carbon.datamap.visible.";
-
-  /**
-   * Fetch and validate the segments.
-   * Used for aggregate table load as segment validation is not required.
-   */
-  public static final String VALIDATE_CARBON_INPUT_SEGMENTS = "validate.carbon.input.segments.";
 
+  private CarbonCommonConstants() {
+  }
   /**
```

Comment:
To make this description more easier to find, suggest to use:
```
 //////////////////////////////////////////////////////////////////////////////////////////
 // System level property start here
 ////////////////////////////////////////////////////////////////////////////////////////// 
 // System level property is the global property for CarbonData 
 // application, these properties are stored in a singleton instance
 // so that all processing logic in CarbonData uses the same 
 // property value 
```

And you can write the description for Table level property and others

Reply:
ok，i modify it